src/widget/tool/screenshotgrabber.cpp

```diff
@@ -192,10 +192,9 @@ void ScreenshotGrabber::chooseHelperTooltipText(QRect rect)
 }
 
 /**
- * @internal
- *
- * Align the tooltip centred at top of screen with the mouse cursor.
- */
+@internal
+@brief Align the tooltip centred at top of screen with the mouse cursor.
```

Comment:
ok, I'm more used to the british style :). Then this doesn't need to be changed.